README.md

```diff
@@ -271,6 +271,26 @@ podman machine ssh
 rpm-ostree override reset --all
 ```
 
+**(Linux only) Unable to create virtual machine:**
+
+When creating a virtual machine on Linux, you may encounter a "macadam binary is missing" or "gvproxy binary is missing" message. This is because the binaries required have to be installed manually.
+
+Follow the below steps for the solution:
```

Comment:
Today someone said something was an 'egg chicken' problem, which I've always known as a 'chicken and egg' problem. I'm not sure who is correct 😆 but apparently we have the same: you always naturally write 'below steps' and I would have gone with 'steps below'. I guess this is fine. ;-)

Reply:
In French it's the egg first :-) https://fr.wikipedia.org/wiki/Paradoxe_de_l%27%C5%93uf_et_de_la_poule
In English the opposite https://en.wikipedia.org/wiki/Chicken_or_the_egg